packages/eas-cli/src/project/publish.ts

```diff
@@ -147,9 +147,11 @@ export async function buildUnsortedUpdateInfoGroupAsync(
 export async function buildBundlesAsync({
   projectDir,
   inputDir,
+  platform = 'all',
 }: {
   projectDir: string;
   inputDir: string;
+  platform?: PublishPlatform | 'all';
```

Comment:
```suggestion
  platform,
}: {
  projectDir: string;
  inputDir: string;
  platform: PublishPlatform;
```
Doesn't _need_ to be optional.